src/main/java/doke/Events.java

```diff
@@ -0,0 +1,54 @@
+package doke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represent the Events task. Extends the Task class.
+ */
+public class Events extends Task{
```

Comment:
Perhaps the name of this class should be `Event` to signify a singular object
```suggestion
public class Event extends Task{
```